homeassistant/components/sensor/vera.py

```diff
@@ -61,11 +61,11 @@ def update(self):
                 self._temperature_units = TEMP_CELSIUS
 
             if self.hass:
-                temp = self.hass.config.temperature(
+                temp = self.hass.config.units.temperature(
                     current_temp,
                     self._temperature_units)
 
-                current_temp, self._temperature_units = temp
```

Comment:
This is a bug. If we convert the temperature we should also overwrite the temperature unit.